src/Conversion/ONNXToTOSA/NN/MaxPoolSingleOut.cpp

```diff
@@ -25,6 +25,44 @@ namespace onnx_mlir {
 
 namespace {
 
+// This calculates the values that need to be added to the padding in order to
+// simulate the ceil mode
+llvm::SmallVector<int64_t> getCeilConstants(int64_t ceil,
+    llvm::ArrayRef<int64_t> inputShape, mlir::ArrayAttr kernelShape,
+    mlir::ArrayAttr strides, mlir::ArrayAttr pads) {
+  // This avoids having more if statements when creating the padding const.
+  if (!ceil) {
```

Comment:
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements